python/Dockerfile.alpine.python3.slim.tpl

```diff
@@ -42,24 +40,14 @@ RUN set -x \
 	&& echo "#{CHECKSUM}" | sha256sum -c - \
 	&& tar -xzf "Python-$PYTHON_VERSION.linux-#{TARGET_ARCH}.tar.gz" --strip-components=1 \
 	&& rm -rf "Python-$PYTHON_VERSION.linux-#{TARGET_ARCH}.tar.gz" \
-	&& mkdir -p /usr/src/python/setuptools \
-	&& curl -SLO https://github.com/pypa/setuptools/archive/v$SETUPTOOLS_VERSION.tar.gz \
-	&& echo "$SETUPTOOLS_SHA256  v$SETUPTOOLS_VERSION.tar.gz" > v$SETUPTOOLS_VERSION.tar.gz.sha256sum \
-	&& sha256sum -c v$SETUPTOOLS_VERSION.tar.gz.sha256sum \
-	&& tar -xzC /usr/src/python/setuptools --strip-components=1 -f v$SETUPTOOLS_VERSION.tar.gz \
-	&& rm -rf v$SETUPTOOLS_VERSION.tar.gz* \
-	&& cd /usr/src/python/setuptools \
-	&& python3 bootstrap.py \
-	&& python3 easy_install.py . \
-	&& mkdir -p /usr/src/python/pip \
-	&& curl -SL "https://github.com/pypa/pip/archive/$PYTHON_PIP_VERSION.tar.gz" -o pip.tar.gz \
-	&& echo "$PYTHON_PIP_SHA256  pip.tar.gz" > pip.tar.gz.sha256sum \
-	&& sha256sum -c pip.tar.gz.sha256sum \
-	&& tar -xzC /usr/src/python/pip --strip-components=1 -f pip.tar.gz \
-	&& rm pip.tar.gz* \
-	&& cd /usr/src/python/pip \
-	&& python3 setup.py install \
-	&& cd .. \
+	&& if [ ! -e /usr/local/bin/pip3 ]; then : \
+		&& curl -SLO "http://resin-packages.s3.amazonaws.com/python/get-pip.py" \
```

Comment:
Can we download the script from here so that we don't have to host it ourselves? https://raw.githubusercontent.com/pypa/get-pip/430ba37776ae2ad89f794c7a43b90dc23bac334c/get-pip.py